content/guides-tutorials/safety.md

```diff
@@ -0,0 +1,103 @@
+
+
+# Sidecar XMP
+
+XMP are XML text files acting as a redundancy of the `library.db` database, which also store the editing history, individually for each picture. They are more easily readable and can be exported, imported and shared with other people, using the [history stack](../module-reference/utility-modules/lighttable/history-stack.md) lighttable module.
+
+XMP sidecars can be enabled or disabled in [preferences > storage > XMP > write sidecar file for each image](../preferences-settings/storage.md). It is strongly recommended to enable them, since they are text files that weigh typically just a couple of kilobytes.
+
+In case you store your original RAW files on an external drive that is shared between several systems, these XMP sidecars will be saved on the external drive and can be use to synchronize the database between all systems (see below, *Using a cloud/NAS with darktable*).
+
+XMP can additionnaly be backed-up too, with external tools like Git.
+
+
+# Using a cloud/NAS with darktable
+
+darktable is not explicitely meant to maintain a server-client architecture since there is no bidirectionnal and dynamic synchronization on, and no way to handle editing conflicts when the same files are edited simultaneously on more than one system. However, it can be used with RAW files on a remote file system shared between several systems, provided users are careful and disciplined enough to not work on the same pictures at the same time on several systems.
+
+---
+
+**Note:** For a more robust versionning and conflict handling, it is advised to use a Git repository in which XMP are commited as plain text and RAW file are commited through `git-annex`.
+
+---
+
+Here is a possible setup for a NAS/cloud hosting of the RAW files:
+
+1. Ensure a systematic folders architecture on the remote storage, with a minimal number of pictures in each sub-folders to avoid synchronizing large amount of data each time. For example: in the main `/NAS/Photos/` directory, each collection of pictures is stored in a sub-folder named `/NAS/Photos/year-month-day-topic/`.
+2. Mount your distant folder as a local drive, using SSH, WebDAV, Samba or any protocol that allows remote access in a way that makes the server locally available to local applications. Ensure the remote location is mounted on a stable and unique mounting point that will be the same in the future, for example `/media/remote/NAS/Photos`.
+3. In darktable preferences, under [import > base directory naming pattern](../preferences-settings/import.md#session_options), input this exact mounting point for all the systems that may connect to this remote storage, for example `/media/remote/NAS/Photos`.
```

Comment:
I thought that this preference setting was only pertinent for importing images from a tethered camera, which you haven't mentioned elsewhere.